setup.py

```diff
@@ -35,22 +35,18 @@
 with open(os.path.join(PROJECT_PATH, 'pyro', '_version.py'), 'w') as f:
     f.write(VERSION.format(version))
 
-# Convert README.md to rst for display at https://pypi.python.org/pypi/pyro-ppl
-# When releasing on pypi, make sure pandoc is on your system:
-# $ brew install pandoc          # OS X
-# $ sudo apt-get install pandoc  # Ubuntu Linux
+
+# READ README.md for long description on PyPi.
+# This requires uploading via twine, e.g.:
+# $ python setup.py sdist bdist_wheel
+# $ twine upload --repository-url https://test.pypi.org/legacy/ dist/*  # test version
+# $ twine upload dist/*
 try:
-    import pypandoc
-    long_description = pypandoc.convert('README.md', 'rst')
-    print(long_description)
+    long_description = open('README.md', encoding='utf-8').read()
 except Exception as e:
-    sys.stderr.write('Failed to convert README.md to rst:\n  {}\n'.format(e))
+    sys.stderr.write('Failed to read README.md\n'.format(e))
     sys.stderr.flush()
-    long_description = open('README.md').read()
-
-# Remove badges since they will always be obsolete.
```

Comment:
You'll want to keep the badge removal bit. But I think we need to update to "first 12 lines"

Reply:
We found that badges seem to work fine, at least with markdown. e.g. here's the numpyro pypi description - https://pypi.org/project/numpyro/ with the badges intact.

Reply:
The problem is not that badges don't show up, the problem is that "they will always be obsolete". Pypi packages are static historical artifacts, whereas those badges are dynamic. If your build happens to be failing at the time of Pypi upload, then that artifact will forever print "build failing" (this has actually happened to me).

Reply:
but I don't care so much. oh well.

Reply:
I see. Thanks for explaining, let me fix that.